js/client-chat.js

```diff
@@ -1115,13 +1124,15 @@
 				case 'highlight':
 				case 'hl':
 					this.add('Set up highlights:');
-					this.add('/highlight add, [word] - Add the word [word] to the highlight list.');
-					this.add('/highlight roomadd, [word] - Add the word [word] to the highlight list of whichever room you used the command in.');
+					this.add('/highlight add [word 1, word 2, ...] - Add the provided list of words to your highlight list.');
```

Comment:
```suggestion
					this.add('/highlight add [word 1], [word 2], [...] - Add the provided list of words to your highlight list.');
```

I think this makes it clearer that you're actually supposed to type the comma if you have multiple words.